Alignment/OfflineValidation/test/submitAllJobs.py

```diff
@@ -0,0 +1,817 @@
+            if "local"+section.title() in self.sections():
+                for option in self.options( "local"+section.title() ):
+                    result[option] = self.get( "local"+section.title(),option )
+        except ConfigParser.NoSectionError, section:
+            msg = ("%s in configuration files. This section is mandatory."
+                   %(str(section).replace(":", "", 1)))
+            #raise AllInOneError(msg)
+        return result     
+
+    ##############################################
+    def getResultingSection( self, section, defaultDict = {}, demandPars = [] ):
+        result = copy.deepcopy(defaultDict)
+        for option in demandPars:
+            try:
+                result[option] = self.get( section, option )
+            except ConfigParser.NoOptionError, globalSectionError:
+                globalSection = str( globalSectionError ).split( "'" )[-2]
+                splittedSectionName = section.split( ":" )
+                if len( splittedSectionName ) > 1:
+                    localSection = ("local"+section.split( ":" )[0].title()+":"
+                                    +section.split(":")[1])
+                else:
+                    localSection = ("local"+section.split( ":" )[0].title())
+                if self.has_section( localSection ):
+                    try:
+                        result[option] = self.get( localSection, option )
+                    except ConfigParser.NoOptionError, option:
+                        msg = ("%s. This option is mandatory."
+                               %(str(option).replace(":", "", 1).replace(
+                                   "section",
+                                   "section '"+globalSection+"' or", 1)))
+                        #raise AllInOneError(msg)
+                else:
+                    msg = ("%s. This option is mandatory."
+                           %(str(globalSectionError).replace(":", "", 1)))
+                    #raise AllInOneError(msg)
+        result = self.__updateDict( result, section )
+        #print result
+        return result
+
+##### method to parse the input file ################################
+def ConfigSectionMap(config, section):
+    the_dict = {}
+    options = config.options(section)
+    for option in options:
+        try:
+            the_dict[option] = config.get(section, option)
+            if the_dict[option] == -1:
+                DebugPrint("skip: %s" % option)
+        except:
+            print("exception on %s!" % option)
+            the_dict[option] = None
+    return the_dict
+
+###### method to create recursively directories on EOS #############
+def mkdir_eos(out_path):
+    newpath='/'
+    for dir in out_path.split('/'):
+        newpath=os.path.join(newpath,dir)
+        # do not issue mkdir from very top of the tree
+        if newpath.find('test_out') > 0:
+            p = subprocess.Popen(["cmsMkdir",newpath], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
actualy `cmsMkdir` still works without complaining (but I'll change it to `eos mkdir` for sake of good practice